drivers/soc/atmel/soc.c

```diff
@@ -70,11 +70,11 @@ static const struct at91_soc __initconst socs[] = {
 #ifdef CONFIG_SOC_SAM9X60
 	AT91_SOC(SAM9X60_CIDR_MATCH, SAM9X60_EXID_MATCH, "sam9x60", "sam9x60"),
 	AT91_SOC(SAM9X60_CIDR_MATCH, SAM9X60D5M_EXID_MATCH,
```

Comment:
I'd like that the #defines are like the other ones: SAM9X60_D5M_EXID_MATCH

Reply:
@noglitch : EXID_MATCH definitions changed with commit feff97a0e7052ec8d397d953d85424a15f25fb87.